seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMySqlSaveModeCatalogIT.java

```diff
@@ -0,0 +1,212 @@
+            };
+
+    void initContainer() throws ClassNotFoundException {
+        // ============= mysql
+        DockerImageName imageName = DockerImageName.parse(MYSQL_IMAGE);
+        mysql_container =
+                new MySQLContainer<>(imageName)
+                        .withUsername(MYSQL_USERNAME)
+                        .withPassword(MYSQL_PASSWORD)
+                        .withDatabaseName(MYSQL_DATABASE)
+                        .withNetwork(NETWORK)
+                        .withNetworkAliases(MYSQL_CONTAINER_HOST)
+                        .withExposedPorts(MYSQL_PORT)
+                        .waitingFor(Wait.forHealthcheck())
+                        .withLogConsumer(
+                                new Slf4jLogConsumer(DockerLoggerFactory.getLogger(MYSQL_IMAGE)));
+        mysql_container.setPortBindings(
+                Lists.newArrayList(String.format("%s:%s", MYSQL_PORT, 3306)));
+
+        Startables.deepStart(Stream.of(mysql_container)).join();
+    }
+
+    @Override
+    @BeforeAll
+    public void startUp() throws Exception {
+        initContainer();
+        initializeJdbcTable();
+    }
+
+    static JdbcUrlUtil.UrlInfo MysqlUrlInfo =
+            JdbcUrlUtil.getUrlInfo("jdbc:mysql://localhost:3308/auto?useSSL=false");
+
+    @TestTemplate
+    public void testCatalog(TestContainer container) throws IOException, InterruptedException {
+        TablePath tablePathMySql = TablePath.of("auto", "mysql_auto_create");
+        TablePath tablePathMySql_Sink = TablePath.of("auto", "mysql_auto_create_sink");
+        MySqlCatalog mySqlCatalog = new MySqlCatalog("mysql", "root", MYSQL_PASSWORD, MysqlUrlInfo);
+        mySqlCatalog.open();
+        CatalogTable catalogTable = mySqlCatalog.getTable(tablePathMySql);
+        // sink tableExists ?
+        boolean tableExistsBefore = mySqlCatalog.tableExists(tablePathMySql_Sink);
+        Assertions.assertFalse(tableExistsBefore);
+        // create table
+        mySqlCatalog.createTable(tablePathMySql_Sink, catalogTable, true);
+        boolean tableExistsAfter = mySqlCatalog.tableExists(tablePathMySql_Sink);
+        Assertions.assertTrue(tableExistsAfter);
+        // isExistsData ?
+        boolean existsDataBefore = mySqlCatalog.isExistsData(tablePathMySql_Sink);
+        Assertions.assertFalse(existsDataBefore);
+        // insert one data
+        mySqlCatalog.executeSql(tablePathMySql_Sink, customSql);
+        boolean existsDataAfter = mySqlCatalog.isExistsData(tablePathMySql_Sink);
+        Assertions.assertTrue(existsDataAfter);
+        // truncateTable
+        mySqlCatalog.truncateTable(tablePathMySql_Sink, true);
+        Assertions.assertFalse(mySqlCatalog.isExistsData(tablePathMySql_Sink));
+        // drop table
+        mySqlCatalog.dropTable(tablePathMySql_Sink, true);
+        Assertions.assertFalse(mySqlCatalog.tableExists(tablePathMySql_Sink));
+        mySqlCatalog.close();
```

Comment:
This is not a e2e case? this will not trigger a ST job, it only test the MySqlCatalog.